<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResourceRestructuringDialog</class>
 <widget class="QWidget" name="ResourceRestructuringDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="param_groupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>基带配置</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>主备：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="name">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>26</height>
           </size>
          </property>
          <property name="maxLength">
           <number>40</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>描述：</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="workMode"/>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="mainStandby"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>工作模式：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>名称：</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPlainTextEdit" name="descs"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_4">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTabWidget" name="tabwidget">
          <property name="minimumSize">
           <size>
            <width>650</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okBtn">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>确认</string>
       </property>
       <property name="iconSize">
        <size>
         <width>14</width>
         <height>14</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editBtn">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>修改</string>
       </property>
       <property name="iconSize">
        <size>
         <width>14</width>
         <height>14</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
       <property name="iconSize">
        <size>
         <width>14</width>
         <height>14</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
